### ISSUE TITLE
documentdb create does not give additional help on next steps

### ISSUE THREAD

**Issue by dondr** [Azure/azure-cli-shell#50]
In the shell type documentdb create and note no additional help is provided on next steps to create this resource.

**Comment by oakeyc** [comment]
![screen shot 2017-04-10 at 1 46 16 pm](https://cloud.githubusercontent.com/assets/16140724/24881935/3f616cca-1df4-11e7-9276-810bc31ef5f0.png)

Is this what you saw? Or are you referring to something else?

**Comment by dondr** [comment]
Yes, what comes next as there is no details on what flags or requirement are next

**Comment by oakeyc** [comment]
Are you referring to something like #22 ?
